Archiver.py

```diff
@@ -69,9 +75,42 @@ def unzip_string(processed_string):
     return uncompressed_string
 
 
+def menu():
+    menu_file_status = "File status: no file selected"
+    menu_main = "Select an option:"
```

Comment:
Why do we need different variables for menu_main and menu_text?

Reply:
United them and rewrite with use first comment